### PR TITLE
Add color params to RouteBonus

### PR DESCRIPTION
**Pull request by Galatolol**
As always, I needed to change colour of an element and it wasn't yet supported.
Updated atom examples.

### CODE REVIEW

File: src/atoms/RouteBonus.jsx [hunk 1]
```diff
@@ -2,26 +2,29 @@ import React from "react";
...
+            fill={fillColor}
+            stroke={strokeColor}
             strokeWidth="1"
           />
           <text
             fontWeight="bold"
             fontSize={size}
             fontFamily="sans-serif"
-            fill={p("black")}
+            fill={textColor}
```
Comment (kelsin) [inline src/atoms/RouteBonus.jsx]:
```suggestion
            fill={c(textColor)}
```

**Comment by kelsin** [comment]
@Galatolol are you able to update this to use the color functions?

File: src/atoms/RouteBonus.jsx (hunk 1 above)
Comment (Galatolol) [reply src/atoms/RouteBonus.jsx]: Done. I also changed the atom example (no need to thave 3 of them).

**Comment by Galatolol** [comment]
You mean that `c()`? Should I do that with `fillColor` and `strokeColor` as well?

**Comment by kelsin** [comment]
Yes! Otherwise they won't use theme colors and will use default browser "black", "red" and "white".

**Comment by kelsin** [comment]
Got it, easier just to do it now that I'm back at a PC.

**Comment by Galatolol** [comment]
Thanks, I had a busy week and kept forgetting about it.

**Pull request merged**